Dockerfile

```diff
@@ -1,62 +1,52 @@
-# The official python:2.7 image no longer receives automatic rebuilds (it's
-# a year old as of October, 2021), so use the latest LTS release of Ubuntu
-# that includes Python 2.7 instead.
-FROM ubuntu:20.04
-
-# Install packages needed to run your application (not build deps).
-RUN set -x \
-    && RUN_DEPS=" \
-    ca-certificates \
-    git \
-    libpq5 \
-    make \
-    postgresql-client \
-    python2.7 \
-    " \
-    && apt-get update && apt-get install -y --no-install-recommends $RUN_DEPS \
+# pull official base image
+FROM python:3.8-slim-bullseye
+
+# set work directory
+WORKDIR /code
+
+# set environment varibles
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+# install deb packages
+RUN apt-get update \
+    && apt-get install --assume-yes --no-install-recommends \
+        git \
+        make \
+        postgresql-client \
```

Comment:
Optional: It'd be nice to install the version of `postgresql-client` matching the server (currently `postgresql-client-14`) via the [PGDG](https://wiki.postgresql.org/wiki/Apt) repo.

Reply:
I've updated the docker image to use PGDG (it took a bit of effort, but I think I finally got it) 👍🏻